### PR TITLE
Create member related models

### PR DESCRIPTION
**Pull request by AnonymousX86**

### CODE REVIEW

**Review by erik-thorsell — CHANGES_REQUESTED** [review]
Please add documentation for your changes.

**Review by ArjunSharda — APPROVED** [review]
LGTM! Waiting for Erik's approval before merge.

**Comment by erik-thorsell** [comment]
We truly appreciate the time and effort you've invested in adding the requested features. Since we have already implemented these changes independently, we will be closing the pull request. Nevertheless, we value your effort and encourage you to continue contributing to the project in the future. Thank you for your participation!

**Pull request closed without merging**
